.github/CODEOWNERS

```diff
@@ -29,3 +29,6 @@ packages/plugins/analytics                                            @yoannmoin
 
 # Custom Hooks
 packages/plugins/custom-hooks                                         @yoannmoinet
+
+# Synthetics
+packages/plugins/synthetics                                           @yoannmoinet @etnbrd
```

Comment:
You need to invite etienne I think:

<img width="659" alt="image" src="https://github.com/user-attachments/assets/fcd85b9d-66c0-4ae9-b6c6-a7cc243c2224" />

Reply:
Good catch.
I'll wait for his review, see if I should instead maybe use a team (cc @etnbrd)

Reply:
You should definitely use the @DataDog/synthetics-ct team, indeed.